conf/app.transport.routes

```diff
@@ -15,3 +15,8 @@ GET        /inland-mode-of-transport/add/:departureId                     contro
 POST       /inland-mode-of-transport/add/:departureId                     controllers.transport.AddInlandModeOfTransportYesNoController.onSubmit(departureId: String, mode: Mode = NormalMode)
 GET        /inland-mode-of-transport/change-add/:departureId              controllers.transport.AddInlandModeOfTransportYesNoController.onPageLoad(departureId: String, mode: Mode = CheckMode)
 POST       /inland-mode-of-transport/change-add/:departureId              controllers.transport.AddInlandModeOfTransportYesNoController.onSubmit(departureId: String, mode: Mode = CheckMode)
+
+GET        /inland-mode-of-transport/:departureId                         controllers.transport.InlandModeController.onPageLoad(departureId: String, mode: Mode = NormalMode)
+POST       /inland-mode-of-transport/:departureId                         controllers.transport.InlandModeController.onSubmit(departureId: String, mode: Mode = NormalMode)
+GET        /inland-mode-of-transport/:departureId                         controllers.transport.InlandModeController.onPageLoad(departureId: String, mode: Mode = CheckMode)
+POST       /inland-mode-of-transport/:departureId                         controllers.transport.InlandModeController.onSubmit(departureId: String, mode: Mode = CheckMode)
```

Comment:
Identical routes? Do these need to have `change` in them or something similar? Are there any other instances of this?